### PR TITLE
auto for smoke case OCP-10658

### PR DESCRIPTION
**Pull request by zhouying7780**
Auto for smoke case: OCP-10658. 
Checkpoint:  `oc rsh` command with specific flags should work well.

### CODE REVIEW

**Comment by adambkaplan** [comment]
Workloads team is in a better position to review.

/assign @damemi

File: test/extended/testdata/cli/pod-with-two-containers.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: doublecontainers
+  labels:
+    name: hello-openshift
+spec:
+  containers:
+  - name: hello-openshift
+    image: aosqe/hello-openshift
```
Comment (soltysh) [inline test/extended/testdata/cli/pod-with-two-containers.yaml]: I'd prefer we stick what's already available and used everywhere so `openshift/hello-openshift` please.

File: test/extended/testdata/cli/pod-with-two-containers.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      protocol: TCP
+    resources:
+      limits:
+        memory: 256Mi
+    terminationMessagePath: "/dev/termination-log"
+    imagePullPolicy: IfNotPresent
+    capabilities: {}
+    securityContext: {}
+  - name: hello-openshift-fedora
+    image: aosqe/fedora_base
```
Comment (soltysh) [inline test/extended/testdata/cli/pod-with-two-containers.yaml]: Ditto, why not just running hello-openshift twice?

Comment (soltysh) [reply test/extended/testdata/cli/pod-with-two-containers.yaml]: openshift or centos orgs are preferred if you look through other test data.

Comment (zhouying7780) [reply test/extended/testdata/cli/pod-with-two-containers.yaml]: Thanks very much, I've updated.

**Comment by zhouying7780** [comment]
@soltysh  Thanks for feedback, I'v updated the testdata, Please help review again.

**Pull request merged**